src/chaplin/views/layout.coffee

```diff
@@ -97,7 +97,7 @@ module.exports = class Layout extends View
   openLink: (event) =>
     return if utils.modifierKeyPressed(event)
 
-    el = if Backbone.$ then event.currentTarget else event.delegateTarget
+    el = event.currentTarget
```

Comment:
Had to use currentTarget because delegateTarget didn't seems to have the same semantics in NativeView than its jQuery counterpart. Mind an issue in the NativeView repo?